platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/ConsensusImpl.java

```diff
@@ -477,6 +485,8 @@ private void voteInAllElections(@NonNull final EventImpl votingWitness) {
                     // no need to vote anymore until we create elections for the next round
                     return;
                 }
+            } else {
+                markerFileWriter.writeMarkerFile(NO_SUPER_MAJORITY_MARKER_FILE);
```

Comment:
Leaving a comment here, to consider whether this is the correct place for this to be written

Looking at this method, the `false` case of this block doesn't *seem* to me to be something that ought to constitute an error. But I'm not familiar enough with this code to say one way or another. Maybe @lpetrovic05  can weigh in

Reply:
yeah, this is the wrong place, this will actually happen all the time until we decide a witness

Reply:
I moved the check to the location where we construct the `ConsensusRound`.